### ISSUE TITLE
connected but no internet access

### ISSUE THREAD

**Issue by jiangfubang** [P0cL4bs/wifipumpkin3#195]
I can't connect to this wifi, I've been trying for days, the phone says "connected but no internet access", then it automatically disconnects, looping
1. I use a brand new and latest kali-2022.2 virtual machine, NAT mode
2. The latest wifipumpkin3 version 1.1.3-R3
The execution command is as follows:
apt install libssl-dev libffi-dev build-essential
apt install hostapd
git clone https://github.com/P0cL4bs/wifipumpkin3.git
cd wifipumpkin3
python3 setup.py install
wifipumpkin3
set interface wlan0
set ssid demo
set proxy noproxy
ignore pydns_server
start

I think this is the most basic build step, but it keeps not working
![1657078197525](https://user-images.githubusercontent.com/30563770/177462254-70f16e38-74d7-4b65-810a-ecaa9396272e.png)

**Comment by mh4x0f** [comment]
Hi, this bug can be come from the program your using for emulate the Kali, I supposition you have running the VirtualBox, my recommendation is using the Vmware (after this test install and test it ) . But using the same configuration with noproxy try to change the dhcp to another IP class using the command bellow:

``` sh
wp3 > dhcpconf 1
``` 
 if not working try with `dhcpconf 2` .

Okay, if still the problem `no connection` try to change to `Bridge` mode this is can solve the problem with Virtual-box. 

The last option is check if iptables roles, before start the wp3 execute this command bellow and send me the output:

```
sudo iptables -L
```

**Comment by harryronhermionebest** [comment]
I have the same problem. when my dummy device connected to the access point, the captive page never shows up. i'm not running the program on a VM. when executing "dhcpconf 1" and "dhcpconf 2" the console returned nothing. When executing "sudo iptables -L" it returns:

└─$ sudo iptables -L          
# Warning: iptables-legacy tables present, use iptables-legacy to see them
Chain INPUT (policy ACCEPT)
target     prot opt source               destination         

Chain FORWARD (policy ACCEPT)
target     prot opt source               destination         

Chain OUTPUT (policy ACCEPT)
target     prot opt source               destination

**Comment by jepunband** [comment]
same issue. installed it on Kali using sudo apt install wifipumpkin3. It is connected to the AP but shows as no internet. i'm not using VM.

**Comment by jiangfubang** [comment]
me too! Modified `dhcpconf 1`, but still not working
$ sudo iptables -L
Warning: iptables-legacy tables present, use iptables-legacy to see them
Chain INPUT (policy ACCEPT)
target prot opt source destination

Chain FORWARD (policy ACCEPT)
target prot opt source destination

Chain OUTPUT (policy ACCEPT)
target prot opt source destination**_

THANK YOU FOR YOUR HELP.

**Comment by jepunband** [comment]
the dhcpconf is fine without changing it in the first place, my private connection is class C ip address and wifipumpkin gives class A ip address. So, not conflicting and by right it should work.
Not sure what else to check.

**Comment by N0el4kLs** [comment]
I got the same problem and my situation is that I can't find the SSID I created just now after one minute later. And I run the `ap` command, it shows running status.
![image](https://user-images.githubusercontent.com/64629436/192222638-2f63bfcb-f6a7-4144-b683-98167870a485.png)
this is the SSID I can find. No SSID called demo I created
![image](https://user-images.githubusercontent.com/64629436/192222716-a81ae98d-849a-49a5-8234-51e54e5f54db.png)
